scripts/install-cd-namespace-resources.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -ue
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ODS_PIPELINE_DIR=${SCRIPT_DIR%/*}
+
+NAMESPACE=""
+RELEASE_NAME="ods-pipeline"
+SERVICEACCOUNT="pipeline"
```

Comment:
Does this service account exists in KinD? If not, perhaps we can use the `default` one?

Reply:
No, but it is overwritten from within the Go tests with `default`, which exists.

Reply:
Cool, then from the Golang tests perspective we're covered. But what if I install the Helm chart in KinD? Wouldn't it fail because of the missing service account?

Reply:
You can always set it via the params. I think this is OK as typically KinD is this throw-away cluster which you only use for running the testsuite. Manual testing I would expect to happen in OpenShift because that is the "real" environment.